lib/bootloader_setup.pm

```diff
@@ -89,13 +89,16 @@ sub bootmenu_default_params {
         type_string " video=hyperv_fb:1024x768";
     }
     type_string_slow "Y2DEBUG=1 ";
-    if (!is_jeos && (check_var('ARCH', 'i586') || check_var('ARCH', 'x86_64'))) {
+
+    return if is_jeos || is_casp('VMX');
```

Comment:
please add a line of comment above here why it does not make sense to continue for jeos or casp.